R/pkg/R/functions.R

```diff
@@ -4426,3 +4426,39 @@ setMethod("timestamp_seconds",
         )
         column(jc)
     })
+
+#' @details
```

Comment:
Hi, @zero323 .
Please move this into to `Window functions` group.
- https://github.com/apache/spark/blob/master/R/pkg/R/functions.R#L3215

Reply:
Thanks for the review @dongjoon-hyun. Adjusted.